### DISCUSSION TITLE
Homebrew binary location

### DISCUSSION THREAD

**Discussion by ghost** [Homebrew/discussions#3775] [Getting started]
Hey,

I am currently trying to configure my SSH client to work with id_ed25519_sk keys.
There are a lot of guides available to make that happened.

My issue is, they all expect that Homebrew installs the binary to `/usr/local/bin/` which seems to be wrong.
After installing openssh via `brew install openssh` there is noting new in `/usr/local/bin/`

So, where does Homebrew put the binaries?

**Comment by carlocab** [comment]
What's the output of `brew --prefix`?

**Comment by ghost** [comment]
The output is `/opt/homebrew`
and there I also find my binaries `/opt/homebrew/bin/`

Thanks!
